client/src/views/home.js

```diff
@@ -34,16 +34,16 @@ export const recentBlocks = ({ t, blocks, loading, ...S }) => homeLayout(
     <div className="blocks-table">
       <div className="blocks-table-row header">
         <div className="blocks-table-cell">{t`Height`}</div>
-        <div className="blocks-table-cell">{t`Timestamp`}</div>
+        <div className="blocks-table-cell">{`${t`Timestamp`} (${formatTimezone(new Date().getTime())})`}</div>
         <div className="blocks-table-cell">{t`Transactions`}</div>
         <div className="blocks-table-cell">{t`Size (KB)`}</div>
         <div className="blocks-table-cell">{t`Weight (KWU)`}</div>
       </div>
       { blocks && blocks.map(b =>
         <div className="blocks-table-link-row">
         <a className="blocks-table-row block-data" href={`block/${b.id}`}>
-          <div className="blocks-table-cell highlighted-text" data-label={t`Height`}>{formatNumber(b.height)}</div>
-          <div className="blocks-table-cell" data-label={t`Timestamp`}>{formatTime(b.timestamp, t)}</div>
+          <div className="blocks-table-cell highlighted-text" data-label={t`Height`}>{b.height}</div>
+          <div className="blocks-table-cell" data-label={t`Timestamp`}>{formatTime(b.timestamp, "",t)}</div>
```

Comment:
The `formatTime` invocation should be `formatTime(b.timestamp, false)`.

The `t` argument is a leftover from when timestamps formatting was locale-dependent and is no longer needed. There are two other places that still pass it, in `views/tx.js` and `views/block.js`, would you mind removing it from there too?

Reply:
changes have been made. Thank you.